exchanges/btse/btse_wrapper.go

```diff
@@ -467,50 +490,22 @@
 // If not specified, all orders of all markets will be cancelled
 func (b *BTSE) CancelAllOrders(orderCancellation *order.Cancel) (order.CancelAllResponse, error) {
 	var resp order.CancelAllResponse
-	markets, err := b.GetSpotMarkets()
+	allOrders, err := b.CancelExistingOrder("", orderCancellation.Pair.String(), "")
 	if err != nil {
-		return resp, err
-	}
-
-	format, err := b.GetPairFormat(orderCancellation.AssetType, false)
-	if err != nil {
-		return resp, err
+		return resp, nil
 	}
 
-	resp.Status = make(map[string]string)
-	for x := range markets {
-		fair, err := b.FormatExchangeCurrency(orderCancellation.Pair,
-			orderCancellation.AssetType)
-		if err != nil {
-			return resp, err
-		}
-
-		checkPair := currency.NewPairWithDelimiter(markets[x].BaseCurrency,
-			markets[x].QuoteCurrency,
-			format.Delimiter).String()
-		if fair.String() != checkPair {
-			continue
-		} else {
-			orders, err := b.GetOrders(checkPair)
-			if err != nil {
-				return resp, err
-			}
-			for y := range orders {
-				success := "Order Cancelled"
-				_, err = b.CancelExistingOrder(orders[y].Order.ID, checkPair)
-				if err != nil {
-					success = "Order Cancellation Failed"
-				}
-				resp.Status[orders[y].Order.ID] = success
-			}
+	for x := range allOrders {
+		if allOrders[x].Status == 6 {
```

Comment:
Would be nice to create an order status type (2 = order inserted, 6 = order cancelled) to rid the use of magic numbers